core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -1283,6 +1284,20 @@ func TestLogPoller_LoadFilters(t *testing.T) {
 		assert.True(t, th.LogPoller.HasFilter("third Filter"))
 		assert.False(t, th.LogPoller.HasFilter("fourth Filter"))
 	})
+
+	t.Run("GetFilters", func(t *testing.T) {
+		filters := th.LogPoller.GetFilters()
+		assert.Equal(t, 3, len(filters))
+		assert.True(t, filters["first Filter"].Name == "first Filter")
+		assert.True(t, reflect.DeepEqual(filters["first Filter"].EventSigs, filter1.EventSigs))
```

Comment:
You don't need to call `reflect.DeepEqual` here, that's already called internally by `assert.Equal`. For example, this should work:

```suggestion
		assert.Equal(t, filter1.EventSigs, filters["first Filter"].EventSigs)
```

Actually, the whole thing could probably be just one line:
```
assert.Equal(t, filter1, filters["first Filter"])
```
If we added Topic2, Topic3, & Topic4 in the original definition of filter1:
```
		Topic2:    evmtypes.HashArray{},
		Topic3:    evmtypes.HashArray{},
		Topic4:    evmtypes.HashArray{},
```
Either way is fine with me

Reply:
Thanks! I just decided to use assert.Equal everywhere to remove the use of `reflect`. I think this should be fine.